src/panels/ShareModelPanel/ShareModel.tsx

```diff
@@ -99,27 +99,42 @@ export default function ShareModel() {
     );
   };
 
+  const updatePermissions = async (
+    action: string,
+    user: string,
+    updatedAccess: any,
+    previousAccess: string | undefined
+  ) => {
+    const response = await setModelSharingPermissions(
+      modelControllerURL,
+      modelUUID,
+      store.getState,
+      user,
+      updatedAccess,
+      previousAccess,
```

Comment:
below you use `permissionTo` and `permissionFrom` those seem like better variable names than these.